src/Integration.Vsix/Analysis/SingleFileMonitor.cs

```diff
@@ -119,23 +119,23 @@ private static void EnsureDirectoryExists(string filePath, IFileSystem fileSyste
             }
         }
 
-        private bool IsDuplicateChangeEvent(string fullPath)
+        private bool IsDuplicateChangeEvent(string path)
```

Comment:
From my understanding only the change event fires multiple times so I separated it into a different method